socorro/scripts/es.py

```diff
@@ -23,7 +24,13 @@
 
 
 def get_conn():
-    cm = ConfigurationManager(ConnectionContext.get_required_config())
+    # Create a configuration manager that will only check the environment for
+    # configuration and not command line parameters
+
+    cm = ConfigurationManager(
+        ConnectionContext.get_required_config(),
+        values_source_list=[environment]
+    )
```

Comment:
This fixes the `es` script so that configman isn't trying to look at `sys.argv` and then getting really sad that it doesn't like what it finds. This fixes it to *only* look at the environment.